numpy/core/_add_newdocs.py

```diff
@@ -4424,6 +4424,11 @@
         since the former invokes ``dtype(None)`` which is an alias for
         ``dtype('float_')``.
 
+    .. versionchanged:: 1.23.0
+       Changing size of dtype no longer possible on F-contiguous arrays. At
+       the same time, only the last axis must be contiguous: the array no
+       longer needs to be C-contiguous.
```

Comment:
I think this belongs near "if ``some_dtype`` has a different number of bytes per entry than the previous dtype" below.

That part of the docstring is now inaccurate.